packages/core/src/common/quick-open-service.ts

```diff
@@ -31,6 +31,7 @@ export namespace QuickOpenOptions {
     export interface Resolved {
         readonly enabled: boolean;
 
+        readonly trim: boolean;
```

Comment:
I won't be able to figure out what it is for, just by looking at a name. It would be good to add docs or come up with a better name.

The best would be to avoid to introduce magic boolean flags for each case and find a way to handle it in the quick command instead.

Reply:
Done.  I renamed this boolean flags and add js-docs.

Reply:
It is the default solution for all cases. To avoid it for quick command, I should add next 
```  
  getOptions(): QuickOpenOptions {
        return { trimInput: false };
    }
```